scripts/apps/authoring/authoring/services/AuthoringService.js

```diff
@@ -248,6 +251,35 @@ export function AuthoringService($q, $location, api, lock, autosave, confirm, pr
         return promise;
     };
 
+    /**
+     * Generate item annotations field
+     *
+     * @param {Object} item
+     */
+    this.generateAnnotations = (item) => {
```

Comment:
Avoid using the internals of how annotations are implemented. Get them using the interface as it's done in `addCommentsForServer`